cpp/ps0000/p0056.so.cpp.cpp

```diff
@@ -1,10 +1,11 @@
-#include <iostream>
+#include <iostream> // 2/2
 
 using namespace std;
 
-int main()
+int main() // 1/1
 {
-  cout << "hello, world" <<"\n" ;
+  cout << "Hello, World\n"; // 3/4
 
-  return 0;
+  return 0; // 1/1
 }
+// great work 😎 9/10 ⭐

```

Comment:
- [x] most of code errors and bugs are fixed
- [x] file is in the right path
- [x] +1 bonus marks for your first commit